examples/classification/plot_digits_classification.py

```diff
@@ -102,3 +102,26 @@
 print(f"Confusion matrix:\n{disp.confusion_matrix}")
 
 plt.show()
+
+###############################################################################
+# We can also build a :func:`~sklearn.metrics.classification_report`
+# through the :ref:`confusion matrix <confusion_matrix>`.
+
+
+def generate_gt_pred_from_cm(cm):
```

Comment:
Nit Pick - The rest of the examples in this file do not create a function. So I think we should follow the same methodology.

Reply:
I pondered that but thought it would make the code clearer, but a simple comment can do the trick.